www/generate_tutorial/src/input/tutorial.md

```diff
@@ -1793,14 +1855,16 @@ Earlier we saw how a function which accepts an open union must account for more
 
 So if I have an `[Ok Str]*` value, I can pass it to functions with any of these types (among others):
 
-- `[Ok Str]* -> Bool`
-- `[Ok Str] -> Bool`
-- `[Ok Str, Err Bool]* -> Bool`
-- `[Ok Str, Err Bool] -> Bool`
-- `[Ok Str, Err Bool, Whatever]* -> Bool`
-- `[Ok Str, Err Bool, Whatever] -> Bool`
-- `Result Str Bool -> Bool`
-- `[Err Bool, Whatever]* -> Bool`
+|              Function Type              | Can it receive `[Ok Str]*`? |
+| :-------------------------------------- | :-------------------------: |
+|           `[Ok Str]* -> Bool`           |             Yes             |
+|           `[Ok Str] -> Bool`            |             Yes             |
+|      `[Ok Str, Err Bool]* -> Bool`      |             Yes             |
+|      `[Ok Str, Err Bool] -> Bool`       |             Yes             |
+| `[Ok Str, Err Bool, Whatever]* -> Bool` |             Yes             |
+| `[Ok Str, Err Bool, Whatever] -> Bool`  |             Yes             |
+|        `Result Str Bool -> Bool`        |             Yes             |
+|     `[Err Bool, Whatever]* -> Bool`     |             Yes             |
```

Comment:
It feels a bit strange to have a column with all `Yes`, especially because the information in that column is already given in the sentence. Did you have a particular reason to create the `Yes` column?

Reply:
It was previously a long list of bullet points. It could stay that way, but I thought this looked nicer. Agree it is a bit strange, but also in context it is showing how all of those functions can receive that input, which is probably surprising for someone new to Roc's type system, and conveys the intended message clearly.